pycc/lccwfn.py

```diff
@@ -0,0 +1,476 @@
+        Fae_ij Lovvv_ij
+        Fme_ij (Fme_im) Loovv_ij
+        Wmbej_ijim ERIovvv_ij, ERIoovv_ij, Loovv_ij
+        Wmbje_ijim (Wmbie_ijmj) ERIovvv_ij, ERIoovv_ij
+        r1_ii Lovvo_ij, ERIovvo_ij
+        r2_ij ERIovvo_ij, ERIovov_ij, ERIvvvo_ij
+        """
+        o = self.o
+        v = self.v
+        F = self.H.F
+        L = self.H.L
+        ERI = self.H.ERI
+
+        Fae_ij = []
+        Fme_ij = []
+
+        Wmbej_ijim = []
+        Wmbje_ijim = []
+        Wmbie_ijmj = []
+        Zmbij_ij = []
+
+        r1_ii = []
+        r2_ij = []
+
+        Fae_ij = self.build_lFae(Fae_ij, self.Local.Fvv_ij, self.Local.Fov_ij, 
+        self.Local.Lovvv_ij, self.Local.Loovv_ij, t1_ii, t2_ij)
+        lFmi = self.build_lFmi(o, F, self.Local.Fov_ij, self.Local.Looov_ij, self.Local.Loovv_ij, t1_ii, t2_ij)
+        Fme_ij = self.build_lFme(Fme_ij, self.Local.Fov_ij, self.Local.Loovv_ij, t1_ii)
+        lWmnij = self.build_lWmnij(o, ERI, self.Local.ERIooov_ij, self.Local.ERIoovo_ij, 
+        self.Local.ERIoovv_ij, t1_ii, t2_ij)
+        Zmbij = self.build_lZmbij(Zmbij_ij, self.Local.ERIovvv_ij, t1_ii, t2_ij)
+        Wmbej_ijim = self.build_lWmbej(Wmbej_ijim, self.Local.ERIoovv_ij, self.Local.ERIovvo_ij, 
+        self.Local.ERIovvv_ij,self.Local.ERIoovo_ij, self.Local.Loovv_ij, t1_ii, t2_ij)
+        Wmbje_ijim, Wmbie_ijmj = self.build_lWmbje(Wmbje_ijim, Wmbie_ijmj, self.Local.ERIovov_ij, 
+        self.Local.ERIovvv_ij,self.Local.ERIoovv_ij, self.Local.ERIooov_ij, t1_ii, t2_ij)
+
+        r1_ii = self.lr_T1(r1_ii, self.Local.Fov_ij , self.Local.ERIovvv_ij, self.Local.Lovvo_ij, self.Local.Loovo_ij, 
+        t1_ii, t2_ij,Fae_ij, Fme_ij, lFmi)
+        r2_ij = self.lr_T2(r2_ij, self.Local.ERIoovv_ij, self.Local.ERIvvvv_ij, self.Local.ERIovvo_ij, self.Local.ERIovoo_ij, 
+        self.Local.ERIvvvo_ij,self.Local.ERIovov_ij, t1_ii, t2_ij, Fae_ij ,lFmi,Fme_ij, lWmnij, Zmbij_ij, Wmbej_ijim, Wmbje_ijim, Wmbie_ijmj)
+
+        return r1_ii, r2_ij    
+
+    def build_lFae(self, Fae_ij, Fvv_ij,Fov_ij, Lovvv_ij, Loovv_ij, t1_ii, t2_ij):
+        #self.fae_t.start()
+        o = self.o
+        v = self.v
+        QL = self.QL
+
+        for ij in range(self.no*self.no):
+            i = ij // self.no
+            j = ij % self.no
+
+            Fae = Fvv_ij[ij].copy()
+
+            Fae_3 = np.zeros_like(Fae)
+            for m in range(self.no):
+                for n in range(self.no):
+                    mn = m *self.no +n
+                    nn = n*self.no + n
+
+                    Sijmn = QL[ij].T @ QL[mn]
+
+                    tmp2 = Sijmn @ t2_ij[mn]
+                    tmp3_0 = QL[ij].T @ self.H.L[m,n,v,v]
+                    tmp3_1 = tmp3_0 @ QL[mn]
+                    Fae_3 -= tmp2 @ tmp3_1.T
+
+            Fae_ij.append(Fae + Fae_3)
+        #self.fae_t.stop()
+        return Fae_ij
+
+    def build_lFmi(self, o, F, Fov_ij, Looov_ij, Loovv_ij, t1_ii, t2_ij):
```

Comment:
Many of the comments I've added for `build_lFae` apply to the other functions below.